substrafl/strategies/fed_avg.py

```diff
@@ -206,6 +213,7 @@ def _perform_local_updates(
         current_aggregation: Optional[SharedStateRef],
         round_idx: int,
         aggregation_id: str,
+        clean_models: bool,
```

Comment:
could you update the docstrings everywhere please ? :)

Reply:
Oh yes, I forgot these